src/main/resources/assets/immersivepetroleum/manual/en_us/pumpjack.txt

```diff
@@ -1,13 +1,17 @@
 Pumpjack
-Pump, pump it up
+Pump, Pump It Up
 <&pumpjack0>
-The Pumpjack is a multiblock that pumps liquids from deposits situated underneath bedrock once they have been tapped by the <link;immersivepetroleum:derrick;§o§nDerrick§r>. <np>
-The structure is formed by using the Engineering's Hammer on the Heavy Engineering Block on the second layer.
+The Pumpjack is a machine which pumps liquids from wellheads created by the <link;immersivepetroleum:derrick;§o§nDerrick§r>.
+The structure is formed by using the Engineer's Hammer on the heavy engineering block on the second layer.<np>
+Pumpjacks are the workhorse machine of petroleum extraction, producing a slow but steady amount of liquid from their wells by lifting it through a one-way valve to overcome the lack of well pressure. Wells that still have pressure will overcome the pumpjack's valve and will gush liquid.
```

Comment:
- Change "**petroleum**" to "**crude**" or "**crude oil**" or "**oil**"
If you want to be more specific you can, but I'm not your mom.